Source/Orts.Simulation/Simulation/RollingStocks/TrainCar.cs

```diff
@@ -1116,62 +1113,51 @@ public virtual void UpdateTunnelForce()
         {
             if (Train.IsPlayerDriven)   // Only calculate tunnel resistance when it is the player train.
             {
-                if (TunnelResistanceDependent)
+                if (CarTunnelData.FrontPositionBeyondStartOfTunnel.HasValue)
                 {
-                    if (CarTunnelData.FrontPositionBeyondStartOfTunnel.HasValue)
-                    {
+                    float? TunnelStart;
+                    float? TunnelAhead;
+                    float? TunnelBehind;
```

Comment:
Those variables don't seem to be used anymore.

Reply:
Thanks.
Fixed in next commit